### DISCUSSION TITLE
DANDI account still pending approval

### DISCUSSION THREAD

**Discussion by MarleneLi** [dandi/helpdesk#97] [Q&A] [answered]
Hi, my Dandi account is still pending approval, though it's already been 2 business days. Can somebody tell me what to do next? I don't even know how to register and apply again.

**Comment by yarikoptic** [comment]
Thanks for reaching out!  Please tell us more on why you need an account -- note that data in DANDI is available openly, so you don't need an account for accessing it to either study or research.

**Reply by MarleneLi** [reply]
I'm an undergraduate research assistant at the University of Michigan and our lab focuses on neuroscience and is interested in streaming the data of Dandi to our software named Improv, so I need access to Dandihub to read some of the code and documentation. I also need access to upload Dandiset. Thanks!

**Reply by yarikoptic** [reply]
you don't need access to dandihub to stream data (to improv which I can't locate or any other software). All of our code and documentation is publicly available under https://github.com/dandi/ .  Could you please tell us more, e.g. in which lab you are an RA and what data is to be uploaded to DANDI?

**Reply by MarleneLi** [reply]
I'm in Draelos Lab under the Biomedical Engineering Apartment. And also I ask for access mainly because I still cannot access the [DANDI JupyterHub](https://hub.dandiarchive.org/). For data to be uploaded to DANDI, I'm not quite sure about it yet but my professor asked me to look at it for now.

**Reply by yarikoptic** [reply, marked as answer]
@draelos confirmed , account was approved.

**Comment by satra** [comment]
@MarleneLi - we could not add you to our slack channel. i would suggest changing the email address of your github account to your university of michigan address (or letting us know what that is via help@dandiarchive.org).
